Assets/MixedRealityToolkit.SDK/Experimental/HandCoach/Scripts/RotateAroundPoint.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    /// <summary>
+    /// This class provides functionality to rotate the hand hint around a pivot point over time.
+    /// </summary>
+    public class RotateAroundPoint : MonoBehaviour
```

Comment:
Similar to my comment above, it might be good to put this helper into its own namespace so people understand that this is related to hand coach. Something like `Microsoft.MixedReality.Toolkit.Experimental.UI.HandCoach`